### ISSUE TITLE
Double and Triple digit collection numbers obscure right arrow

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/build-a-molecule#147]
**Test device**
Windows 10 Laptop
**Operating System**
Windows 10
**Browser**
Chrome
**Problem description**
Once the collection number reaches the double digits it touches the right collection view arrow. Once the collection number reaches the triple digits it is obscured by the right collection view arrow.
**Steps to reproduce**
1. Add the query parameter ?easyMode
2. Enter the "Single" screen
3. Complete the collections up until collection 11
4. Return to collection 10 and Observe
5. Complete the collections up until collection 101
6. Return to collection 100 and Observe
**Visuals**
![2020-01-23BldMlcle](https://user-images.githubusercontent.com/54685448/73032069-2ed4b900-3dfb-11ea-8d2d-30a6462fbc37.gif)

<details><summary><b>Troubleshooting information:</b></summary>
!!!!! DO NOT EDIT !!!!!
Name: ‪Build a Molecule‬
URL: http://phettest.colorado.edu/build-a-molecule/build-a-molecule_en.html?ea&brand=phet
Version: 0.0.0-dev.47 (require.js)
Features missing: generatedcontent, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36
Language: en-US
Window: 1536x754
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by Denz1994** [comment]
An additional option was added to `NextPreviousNavigationNode` to adjust the arrow padding (https://github.com/phetsims/scenery-phet/commit/af25310bf7688bfcc25d550607755802e3f117c3).

It looks like you had to complete about 100 collections. I added a query parameter to change the labeling on the collections so it starts off at specific collection. It doesn't create `x` amount of collections but it does change the labeling. Try out `?skipLevels={number}`. I hope that makes life easier when testing. 

See this [dev version](https://phet-dev.colorado.edu/html/build-a-molecule/0.0.0-dev.48/phet/build-a-molecule_all_phet.html) for the above changes @loganbraywork.
